scripts/run_seqware_workflow.py

```diff
@@ -284,24 +286,6 @@ def main():
     execute("tar -cvzf {0}/{1}.qc_metrics.tar.gz -C {0} qc_metrics.json".format(output_dir, prefix))
     execute("cat {0}/{1}.qc_metrics.tar.gz | md5sum | cut -b 1-33 > {0}/{1}.qc_metrics.tar.gz.md5".format(output_dir, prefix))
 
-    if args.output_file_basename is not None:
```

Comment:
the file naming does not work in the seqware workflow job itself, let's still do it here in this function.